Application/Setting/Authorization/Account/Service.php

```diff
@@ -153,7 +154,7 @@ public function createAccount(IFormInterface $Form, $Account)
      *
      * @return IFormInterface|string
      */
-    public function changeAccount(IFormInterface $Form, TblAccount $tblAccount, $Account)
+    public function changeAccountForm(IFormInterface $Form, TblAccount $tblAccount, $Account)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method changeAccountForm() has an NPath complexity of 23652. The configured NPath complexity threshold is 200.](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5646354)